src/appleseed/renderer/kernel/lighting/sppm/sppmlightingengine.cpp

```diff
@@ -664,4 +665,89 @@ ILightingEngine* SPPMLightingEngineFactory::create()
             m_params);
 }
 
+Dictionary SPPMLightingEngineFactory::get_params_metadata()
+{
+    Dictionary metadata;
+    add_common_params_metadata(metadata);
+
+    metadata.dictionaries().insert(
+        "enable_caustics",
+        Dictionary()
+            .insert("type", "bool")
+            .insert("default", "false")
+            .insert("help", "Enable caustics"));
+
+    metadata.dictionaries().insert(
+        "photon_type",
+        Dictionary()
+            .insert("type", "enum")
+            .insert("options", "mono|poly")
```

Comment:
Maybe options should be a nested dictionary. That way we could add a description for each value...